docs: document the process for tracking security sensitive issues (#8078)
near/nearcore: docs/practices/security_sensitive_issues.md

> ## Security sensitive issues

Author: We are actually planning on moving away from security@near.org.  But that will take some time.  I suppose I can document that for now and have to remember to update it when it changes.

As nearcore is open source, all of its issues and pull requests are also
publicly tracked on github.  However, from time to time, if a security sensitive
issue is discovered, those cannot be tracked publicly on github.  However, we
should promote as similar a development process to work on such issues as
possible.  To enable this, below is the high level process for working on
security sensitive issues.

1. There is a [private fork of
   nearcore](https://github.com/near/nearcore-private) on github.  Access to
   this repository is restricted to the set of people who are trusted to work on
   and have knowledge about security sensitive issues pertaining to nearcore.

   This repository can manually synced with the public nearcore repository using
   the following commands:

>        git remote add nearcore-public git@github.com:near/nearcore
>        git remote add nearcore-private git@github.com:near/nearcore-private
>        git fetch nearcore-public
>        git push nearcore-private nearcore-public/master:master

Reviewer: ````suggestion
    ```  console
    $ git remote add nearcore-public git@github.com:near/nearcore
    $ git remote add nearcore-private git@github.com:near/nearcore-private
    $ git fetch nearcore-public
    $ git push nearcore-private nearcore-public/master:master
    ```
````

style issues: 

* stick to fenced code blocks
* use `console` as a language and a leading `$` to emphasize that this is an interactive shell command.

2. All security sensitive issues must be created on this repository.  You must
   also assign one of the `[P-S0, P-S1]` labels to the issue to indicate the severity
   of the issue.  The two criteria to use to help you judge the severity are
   ease of carrying out the attack and the impact of the attack.  An attack that
   is easy to do or can have a huge impact should have the `P-S0` label and `P-S1`
   otherwise.

> 3. All security sensitive pull requests should also be created on this
>    repository.  Note that once a PR is approved on the private repository, it
>    should not be merged into the private repository.  Instead it should be first

Reviewer 2: ```suggestion
3. All security sensitive pull requests should also be created on the private
   repository.  Note that once a PR has been approved, it
   should not be merged into the private repository.  Instead it should be first
```
